doc/runtime-information.rst

```diff
@@ -59,12 +59,70 @@ bundled and in the bundle folder if it is bundled::
     bundle_dir = getattr(sys, '_MEIPASS', path.abspath(path.dirname(__file__)))
     path_to_dat = path.abspath(path.join(bundle_dir, 'other-file.dat'))
 
+Or, if you'd rather use pathlib_::
+
+    from pathlib import Path
+    import sys
+
+    if getattr(sys, 'frozen', False) and hasattr(sys, '_MEIPASS'):
+        bundle_dir = Path(sys._MEIPASS)
+    else:
+        bundle_dir = Path(__file__).parent
+
+    path_to_dat = bundle_dir.absolute() / "other-file.dat"
```

Comment:
FTR `Path.absolute()` is not documented and thus not officially supported. See [this thread](https://discuss.python.org/t/pathlib-absolute-vs-resolve/2573/15), it contains useful information.

Reply:
Huh, that's a nasty surprise. Well I'm not sure that thread comes to any usable solution. `Path.resolve()` would not work if `__file__` turns out to be relative because it requires the preceding `./` to actually make it absolute but `Path("./xyz")` automatically normalises  to just `Path("xyz")` which breaks `resolve`. I take it the fix is supposed to be?:
```
path_to_dat = Path.cwd() / bundle_dir / "other-file.dat"
```

Reply:
FWIW, I always used the same code without `.absolute()` and it works, if that [helps](https://github.com/nuxeo/nuxeo-drive/blob/442c1ac2fc18aa52c5208e4717d24361001ff7ce/nxdrive/options.py#L101-L108). Maybe is it "just OK" to go with or without `.absolute()`, I just wanted to show the possible issues for the future if everyone would use that code in all their projects :)

Reply:
Well the danger with not using absolute paths is:
```python
>>> Path(".").parent
WindowsPath('.')
```
Which would mean that all your resources are searched for one folder too deep. But for this to actually happen requires that a) Python hasn't already resolved `__file__` for you and b) your `cwd` is a subfolder inside your app (the app's root will still work fine).

Reply:
Maybe it is good to go with `.absolute()` after all :)
It is sad to not having an `os.abspath()` equivalent for `Path`.

Reply:
Well I don't think we should use unsupported methods in our docs so I think I'll go for the `Path.cwd() / ...` approach. And yes it is sad - it looks like I have quite a bit of refactoring to do...